docs/integrations/context-standards-mandatory.md

```diff
@@ -1,22 +1,12 @@
 ---
-id: context-standards
-title: Context Standards
+id: context-standards-mandatory
```

Comment:
We need to add a redirect to the `_redirects` file

Reply:
added one to context-standards-about